eth/tracers/api_test.go

```diff
@@ -362,19 +362,21 @@ func TestOverridenTraceCall(t *testing.T) {
 			},
 		},
 		// Invalid call without state overriding
-		{
-			blockNumber: rpc.PendingBlockNumber,
-			call: ethapi.CallArgs{
-				From:  &randomAccounts[0].addr,
-				To:    &randomAccounts[1].addr,
-				Value: (*hexutil.Big)(big.NewInt(1000)),
-			},
-			config: &TraceCallConfig{
-				Tracer: &tracer,
-			},
-			expectErr: core.ErrInsufficientFundsForTransfer,
-			expect:    nil,
-		},
+		// NOTE: this test has been disabled, as at core-geth we override
+		// vm.(Can)Transfer in order to have compatible tracers with OE
+		// {
+		// 	blockNumber: rpc.PendingBlockNumber,
+		// 	call: ethapi.CallArgs{
+		// 		From:  &randomAccounts[0].addr,
+		// 		To:    &randomAccounts[1].addr,
+		// 		Value: (*hexutil.Big)(big.NewInt(1000)),
+		// 	},
+		// 	config: &TraceCallConfig{
+		// 		Tracer: &tracer,
+		// 	},
+		// 	expectErr: core.ErrInsufficientFundsForTransfer,
+		// 	expect:    nil,
```

Comment:
Would it make sense to modify the test to `expectErr: nil; expect: <blah>`? (but leaving a comment about the difference, as you already have done)

Reply:
Thought of that too, but this test, wanna test the override functionality. Testing what you described, is being covered already in the tracers tests.